src/components/Loader/index.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface ILoaderProps {
+  variant?: 'circular' | 'bar';
+  progress?:
+    | -1
+    | 0
+    | 1
+    | 2
+    | 3
+    | 4
+    | 5
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 17
+    | 18
+    | 19
+    | 20
+    | 21
+    | 22
+    | 23
+    | 24
+    | 25
+    | 26
+    | 27
+    | 28
+    | 29
+    | 30
+    | 31
+    | 32
+    | 33
+    | 34
+    | 35
+    | 36
+    | 37
+    | 38
+    | 39
+    | 40
+    | 41
+    | 42
+    | 43
+    | 44
+    | 45
+    | 46
+    | 47
+    | 48
+    | 49
+    | 50
+    | 51
+    | 52
+    | 53
+    | 54
+    | 55
+    | 56
+    | 57
+    | 58
+    | 59
+    | 60
+    | 61
+    | 62
+    | 63
+    | 64
+    | 65
+    | 66
+    | 67
+    | 68
+    | 69
+    | 70
+    | 71
+    | 72
+    | 73
+    | 74
+    | 75
+    | 76
+    | 77
+    | 78
+    | 79
+    | 80
+    | 81
+    | 82
+    | 83
+    | 84
+    | 85
+    | 86
+    | 87
+    | 88
+    | 89
+    | 90
+    | 91
+    | 92
+    | 93
+    | 94
+    | 95
+    | 96
+    | 97
+    | 98
+    | 99
+    | 100;
+  flat?: boolean;
+  className?: string;
+}
+
+const TEST_ID = '@fv/Loader';
+
+export const Loader: React.FC<ILoaderProps> = ({ variant = 'circular', flat = false, progress = -1 }) => {
+  const calculateProgress = () => {
```

Comment:
What's the point of `progress === -1`? If I understood correctly if progress === -1 it should infinite animation. Maybe we should show it if there are no progress value?

Reply:
Yes, it was simpler for the storybook as well to just put -1 (the default value is -1 so you wouldn't have to add a progress prop -1 to show the loader).

Reply:
updated!